composer.json

```diff
@@ -27,6 +27,7 @@
             "app/_config.php",
             "app/_config/assets.yml",
             "app/_config/database.yml",
+            "app/_config/email.yml",
```

Comment:
How that file is not in the commit?

Reply:
There's no app folder in this project.  We have silverstripe-installer in composer.json so presumably it gets copied in on deploy

Reply:
It should have been added to `mysite/_config`.  Once a file has been added to `project-files-installed`, it's not  going to try to add it anymore ... so it won't try to add it on deploy.

Reply:
:shrug: probably a wider issue with this setup then since the mysite/_config folder is missing a bunch of the yml files - https://github.com/silverstripe/demo.silverstripe.org/tree/master/mysite/_config